packages/notebook/src/model.ts

```diff
@@ -65,8 +65,7 @@ export class NotebookModel implements INotebookModel {
    */
   constructor(options: NotebookModel.IOptions = {}) {
     this.sharedModel = sharedModels.YNotebook.create({
-      disableDocumentWideUndoRedo: options.disableDocumentWideUndoRedo ?? false,
-      initialCellType: options.defaultCell
+      disableDocumentWideUndoRedo: options.disableDocumentWideUndoRedo ?? false
```

Comment:
Is there a reason to drop the `initialCellType`? As there is a [setting](https://github.com/jupyterlab/jupyterlab/blob/adeae856ffe5a11221d4a3e01a8e2e508ef4ab0a/packages/notebook-extension/schema/tracker.json#L789) to change the default, it seems that it should be kept.

Ok it seems this is gonna be very tricky as the shared model is instantiated independently of the notebook model. But it is nevertheless good to keep it around for now.

Reply:
> Is there a reason to drop the initialCellType? As there is a [setting](https://github.com/jupyterlab/jupyterlab/blob/adeae856ffe5a11221d4a3e01a8e2e508ef4ab0a/packages/notebook-extension/schema/tracker.json#L789) to change the default, it seems that it should be kept.

We don't need it in YNotebook since the YNotebook will never add a cell by itself.

Reply:
Thanks for the reply @hbcarlos

Reply:
Actually, I'm going to revert this change to add a new cell in `YNotebook.deleteCells`. This way, we can reduce the number of times the server has to add the cell, as Kevin commented here https://github.com/jupyterlab/jupyterlab/pull/13141#discussion_r993241370